src/client/Microsoft.Identity.Client/AppConfig/IManagedIdentity.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.AppConfig
+{
+    /// <summary>
+    /// Interface for managed identity configuration supported by MSAL.
+    /// </summary>
+    public interface IManagedIdentity
```

Comment:
Wondering if we can name this something different (IManagedIdentityConfiguration) as another interface IManagedIdentityApplication already exists that looks confusing.